sdk/messaging/azeventhubs/README.md

```diff
@@ -23,41 +28,35 @@ Install the Azure Event Hubs client module for Go with `go get`:
- - ConsumerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewConsumerClient)
- - ProducerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewProducerClient)
+ - ConsumerClient: [link][azeventhubs_serviceprinciple_consumerclient]
+ - ProducerClient: [link][azeventhubs_serviceprinciple_producerclient]
 
 #### Using a connection string
- - ConsumerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewConsumerClientFromConnectionString)
- - ProducerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewProducerClientFromConnectionString)
+ - ConsumerClient: [link][azeventhubs_connectionstring_consumerclient]
+ - ProducerClient: [link][azeventhubs_connectionstring_producerclient]
 
 # Key concepts
 
-An Event Hub [**namespace**](https://docs.microsoft.com/azure/event-hubs/event-hubs-features#namespace) can have multiple event hubs. Each event hub, in turn, contains [**partitions**](https://docs.microsoft.com/azure/event-hubs/event-hubs-features#partitions) which store events.
+An Event Hub [**namespace**][eventhubs_namespace] can have multiple event hubs. Each event hub, in turn, contains [**partitions**][eventhubs_partition] which store events.
 
-Events are published to an event hub using an [event publisher](https://docs.microsoft.com/azure/event-hubs/event-hubs-features#event-publishers). In this package, the event publisher is the [ProducerClient](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#ProducerClient)
+Events are published to an event hub using an [event publisher][eventhubs_eventpublisher]. In this package, the event publisher is the [ProducerClient][azeventhubs_producerclient].
 
-Events can be consumed from an event hub using an [event consumer](https://docs.microsoft.com/azure/event-hubs/event-hubs-features#event-consumers). In this package there are two types for consuming events: 
-- The basic event consumer is the, in the [ConsumerClient](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#ConsumerClient). This consumer is useful if you already known which partitions you want to receive from.
-- A distributed event consumer, which uses Azure Blobs for checkpointing and coordination. This is implemented in the [Processor](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#Processor). This is useful when you want to have the partition assignment be dynamically chosen, and balanced with other Processor instances.
+Events can be consumed from an event hub using an [event consumer][eventhubs_eventconsumer]. In this package there are two types for consuming events: 
+- The basic event consumer is the, in the [ConsumerClient][azeventhubs_consumerclient]. This consumer is useful if you already known which partitions you want to receive from.
```

Comment:
"The basic event consumer is the," ... the what?

Reply:
Also, not to self ... missed "if you already known which".

Reply:
@richardpark-msft Can you address this?

Reply:
```suggestion
- The basic event consumer is the [ConsumerClient][azeventhubs_consumerclient]. This consumer is useful if you already known which partitions you want to receive from.
```

Reply:
@bobtabor-msft, does this edit sound okay?